src/components/ContentWidgets/ToDoList/ToDoListEditor.tsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import { WidgetEditorProps } from '../../ContentMapping/ContentMapping';
+import { ToDoListItem } from './ToDoList';
+
+export const ToDoListEditor: React.FC<WidgetEditorProps> = ({
+    originalContent,
+    editedContent,
+    setEditedContentOnChange
+}) => {
+    let listContent = null;
+
+    // let findItemIndex = (findItemID: number, copyEditedContent: ToDoListItem[]) => {
+        
+    //     for (let i = 0; i < copyEditedContent.length; i++) {
+    //         if (findItemID == copyEditedContent[i].toDoListItem_order) {
+    //             return i;
+    //         }
+    //     }
+    //     return -1;
+    // }
+
+    if (editedContent.toDoList_content != null) {
```

Comment:
You should try to only define these functions once rather than for every single list item. You would achieve this by having functions like `removeAction` or `changeInput` take in an extra index as their parameter.